config/application.rb

```diff
@@ -23,7 +23,7 @@ module ReactAndRails
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-
+    config.hosts << "db8a62fb3a65410586e2601e633b5e52.vfs.cloud9.us-west-2.amazonaws.com"
```

Comment:
What is this all about?

Reply:
Yea, this surprised me as well.  Its a new setting in Rails 6, and I'm working on Cloud 9